lib/array_intersection.rb

```diff
@@ -0,0 +1,5 @@
+# Creates a new array to return the intersection of the two input arrays
+def intersection(array_1, array_2)
+  puts "NOT IMPLEMENTED"
+  return []
```

Comment:
How would you feel about doing `raise NotImplementedError` here instead?

Best practice I think is generally to avoid `puts` statements and other things that will print to the console in the middle of tests because it kind of gets randomly interspersed with the test runner's output.

Reply:
done!